__test__/integration/debt_token_api/runners/transfer_from.ts

```diff
@@ -4,15 +4,28 @@ import { BigNumber } from "bignumber.js";
 // Types
 import { ScenarioRunner } from "./";
 import { DebtTokenScenario } from "../scenarios";
+import { TxData } from "src/types";
 
 // Wrappers
 import { MockERC721ReceiverContract, TokenRegistryContract } from "src/wrappers";
 
+// APIs
+import { DebtTokenAPI } from "src/apis";
+
 // Utils
 import { ACCOUNTS } from "../../../accounts";
 
 const TX_DEFAULTS = { from: ACCOUNTS[0].address, gas: 4712388 };
 
+export interface TransferAPICallParameters {
+    api: DebtTokenAPI;
+    from: string;
+    to: string;
+    tokenID: BigNumber;
+    data: string;
+    txOptions: TxData;
+}
+
 export class TransferFromScenarioRunner extends ScenarioRunner {
```

Comment:
I would move this functionality into an abstract base class that both `transfer` and `transferFrom` runners inherit as opposed to `transfer` extending `transferFrom`